### DISCUSSION TITLE
Guidelines for performant sandboxes?

### DISCUSSION THREAD

**Discussion by andreifloricel** [codesandbox/codesandbox-client#6281] [General]
Hi,

we (at https://adaptabletools.com/) are currently reorganising/rewriting our docs and would really like to move all our demos to Codesandbox.
One impediment is the long initial waiting time.
See this basic sample: https://codesandbox.io/s/stoic-nova-iv89h (more complex projects take even longer).

I found this issue with some recommendations from @CompuIves but it is relatively old: https://github.com/codesandbox/codesandbox-client/issues/1220#issuecomment-432204369

Are the listed recommendations still relevant?
If not, what would be the general recommendations/tips to get a codesandbox as performant as possible?
Maybe it's worth creating such a list of tips&tricks in your docs ;)

Thanks

**Comment by acekyd** [comment]
Hi @andreifloricel, I believe the recommendations are still relevant. Please let us know if you have any issues trying to apply them.

Thank you for the suggestion about having some recommendations, tips and tricks for better performance in the docs. Will pass that on to the rest of the team.
